src/wait-strategy.ts

```diff
@@ -85,6 +87,12 @@ export class LogWaitStrategy extends AbstractWaitStrategy {
       }, startupTimeout);
 
       const comparisonFn: (line: string) => boolean = (line: string) => {
+        const timestamp = new Date(line.split(" ")[0]).getTime();
+        if (!isNaN(timestamp) && timestamp < startedAt) {
```

Comment:
Should this be OR?

Reply:
I went with a conservative approach and if we don't get a valid timestamp at the start of the line, we do still test the line. I thought that this was better than perhaps accidentally never declaring a container as ready.